### PR TITLE
Report size of initramfs correctly

### PR DESCRIPTION
**Pull request by zaneb**
The initramfs is created by concatenating the ignition archive on the
end of the base initramfs. Therefore the length depends on the length of
the ignition data.

Previously we were reporting only the length of the base initramfs, with
the result that the ignition was never included when the image was
served.

### CODE REVIEW

File: pkg/imagehandler/imagefile.go [hunk 1]
```diff
@@ -36,11 +36,26 @@ type imageFile struct {
 var _ fs.File = &imageFile{}
 
 func (f *imageFile) Init(inputFile baseFile) error {
-	if f.imageReader == nil {
-		var err error
-		ignition := &isoeditor.IgnitionContent{Config: f.ignitionContent}
-		f.imageReader, err = inputFile.InsertIgnition(ignition)
+	if f.imageReader != nil {
```
Comment (sadasu) [inline pkg/imagehandler/imagefile.go:39]: Should we also check if f.size is set or non-zero? It is possible that there is a valid f.imageReader but we never got to set f.size correctly. 
If f.imageReader != nil
        if f.size == 0 { 
                attempt to calculate size again
        } else {
                 return nil
        }
}

Comment (zaneb) [reply pkg/imagehandler/imagefile.go:39]: It's set at creation time: https://github.com/openshift/image-customization-controller/blob/main/pkg/imagehandler/imagehandler.go#L102

We should probably get rid of that, but for now let's get it working.

**Pull request merged**